bats/tests/k8s/wasm.bats

```diff
@@ -95,12 +95,17 @@ get_host() {
         local LB_IP
         local output='jsonpath={.status.loadBalancer.ingress[0].ip}'
         LB_IP=$(kubectl get service traefik --namespace kube-system --output "$output")
+        assert_not_equal "$LB_IP" "" || return
         echo "$LB_IP.sslip.io"
     else
         echo "localhost"
     fi
 }
 
+@test 'wait for traefik to get IP' {
+    try get_host >/dev/null
```

Comment:
Why do we need the `>/dev/null`?

Reply:
We don't really need it, it's just not useful in the logs. I can drop the redirect (and have the address show up), I guess? (It's not an issue because that's the WSL IP anyway)

Reply:
Why would it be in the logs? Command output is only in the logs when the test fails. Maybe it won't be useful; but it doesn't hurt either.